packages/core/src/index.ts

```diff
@@ -910,8 +910,14 @@ class Hopr extends EventEmitter {
    * @param destination PeerId of the destination
    * @param intermediatePath optional set path manually
    * @param hops optional number of required intermediate nodes
+   * @returns hex representation of ack challenge
    */
-  public async sendMessage(msg: Uint8Array, destination: PeerId, intermediatePath?: PublicKey[], hops?: number) {
+  public async sendMessage(
+    msg: Uint8Array,
+    destination: PeerId,
+    intermediatePath?: PublicKey[],
+    hops?: number
+  ): Promise<string> {
```

Comment:
Doesn't the `async` keyword in TS cause to automatically wrap the returned type in `Promise` ? Therefore it should be unnecessary.

Reply:
No, the typing still needs to be added manually.